includes/admin/class-wc-admin-dashboard.php

```diff
@@ -226,14 +228,17 @@ private function status_widget_stock_rows() {
 						$nostock
 					)
 				);
-				set_transient( $transient_name, $lowinstock_count, DAY_IN_SECONDS * 30 );
+
+				set_transient( $transient_name, (int) $wpdb->get_var( $lowinstock_count_query ), DAY_IN_SECONDS * 30 );
 			}
 
 			$transient_name   = 'wc_outofstock_count';
 			$outofstock_count = get_transient( $transient_name );
 
 			if ( false === $outofstock_count ) {
-				$outofstock_count = (int) $wpdb->get_var(
+
+				$outofstock_count_query = apply_filters(
+					'woocommerce_status_widget_outofstock_count_query',
```

Comment:
Please update to `woocommerce_status_widget_out_of_stock_count_query`.

Reply:
@claudiosanches changed, Thanks!